client/snapshot.go

```diff
@@ -88,6 +90,21 @@ func (sh *Snapshot) IsValid() bool {
 	return !(sh == nil || sh.SetID == 0 || sh.Snap == "" || sh.Revision.Unset() || len(sh.SHA3_384) == 0 || sh.Time.IsZero())
 }
 
+// ContentHash returns a hash that can be used to identify the snapshot
+// but does not contain data like "time"
+func (sh *Snapshot) ContentHash() ([]byte, error) {
+	sh2 := *sh
+	sh2.SetID = 0
+	sh2.Time = time.Time{}
+	sh2.Auto = false
```

Comment:
this is also annoying (that being auto or not is stored in the files) :/  anyway we need to discuss what to do about expiring snapshots vs imports